pkg/inspector/image-inspector.go

```diff
@@ -123,6 +125,13 @@ func (i *defaultImageInspector) Inspect() error {
 		}
 	}
 
+	packageInspector := content.NewDefaultPackageInspector()
```

Comment:
new scan type.  Also means we probably need to factor the serve portion of the code a little better to be able to serve based on requested types if it is not already doing that.